todo_app/end_to_end_test/test_selenium.py

```diff
@@ -4,8 +4,9 @@
 from threading import Thread
 from selenium import webdriver
 import pymongo
+import time
 
-
+LOGIN_DISABLED=True
```

Comment:
This line doesn't do anything. Maybe you meant `os.environ['LOGIN_DISABLED'] = 'True'`? Then the app will pick it up.

Slightly neater to do that inside the `app_with_temp_board` function though.